### PR TITLE
thisis-mahyar/docs/trello

### PR DESCRIPTION
**Pull request by thisis-mahyar**
:)

### CODE REVIEW

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+Your workspace is divided into two main sections. Although there’s a right sidebar, it mostly duplicates the features in the left section, so we’ll focus on the left side for clarity.
+
+![](photos/05-workspace.jpg)
+
+Starting from the top, you’ll see **Boards**, **Templates**, and **Home**:
+
+- **Boards**: This is the main section where you’ll manage and access your project boards. We’ll dive into the board features in detail later.
+  
+- **Templates**: Here, you can browse a variety of templates organized by category. These templates cover a wide range of project types, including non-technical and non-programming uses.
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:73]: use cases or usages

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+### User Experience
+
+The login process is streamlined and intuitive, making it inviting and easy to sign up or log in. However, one concern is that Trello sometimes skips the password requirement when logging in, which raises privacy and security questions.
+  
+It would be beneficial to remove the **Recently Viewed** section from the workspace. Personally, I think navigation should be straightforward, with a single, clear path to access each board. The multiple ways to navigate to a board can feel redundant and cluttered.
+
+![](photos/02-user-experience.jpg)
+
+## Pricing
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:22]: turn these heading into level three headings and add a heading level 2 above these ones which reads as "Features"

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+#### 1. Workspace panel
+
+Here, we have the workspace panel again, with a few additional options:
+
+![](photos/08-workspace-panel.jpg)
+
+Section | Description
+:- | :-
+*More from Atlassian |  The icon on the far left (a 3x3 grid), which is primarily for advertising other Atlassian products.
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:127]: remove star

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+This section includes options specific to your board:
+
+![](photos/09-board-panel.jpg)
+
+Section | Description
+:- | :-
+Name | Double-click the board name to rename it.
+Star | Star your board to have it appear at the top of your boards list.
+Change visibility | Has already told.
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:143]: has already been discussed

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+Customize views | The default view is the Kanban board, but Premium users can access additional views like Table, Calendar, Timeline, Dashboard, and Map. Only having the Kanban view can be a disadvantage if you’re looking for more layout options, which might make other tools appealing in comparison.
+Power-Ups | Add features and integrations like Google Drive or Slack to your board. Power-Ups work like extensions. E.g., using GitHub power-up, you can attach branches, commits, etc. directly to your cards.
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:145]: etc,

Comment (atareversei) [reply docs/main/03-research/01-vendors/04-trello/04-trello.md:145]: and other GitHub features,

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+Automation | Will be explained later
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:146]: period

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+Filters | Filter cards using options like Members, Due date, Labels, Activity.
+Share | Share your board with others.
+Menu | A menu with options: Description for the board, See the Activity, Comments, Archived Items, Change Workspace, Set Permissions to the Members, Change Background, Automations, Power-Ups, Labels, Stickers, Watch, Email-to-board (add cards via email), Copy board, Print, export, share, Close the board.
+
+#### 3. View
+
+The default view here is a Kanban board, a visual tool for managing workflows and tracking tasks through stages like "To Do," "In Progress," and "Done." It helps teams quickly see the status of tasks and improve task flow.
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:153]: "To Do",

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+![](photos/11-list-actions.jpg)
+
+Option | Description
+:- | :-
+Add card | Add a new card to the list.
+Copy list | Create a duplicate of the list by entering a new name and clicking Create list.
+Move list | Move the list to a different position.
+Move all cards in this list | Transfer all cards to another list.
+Sort by | Organize cards by date, name, etc.
+Automation | Set automation options for this list
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:172]: period

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+Menu | A menu with options: Description for the board, See the Activity, Comments, Archived Items, Change Workspace, Set Permissions to the Members, Change Background, Automations, Power-Ups, Labels, Stickers, Watch, Email-to-board (add cards via email), Copy board, Print, export, share, Close the board.
+
+#### 3. View
+
+The default view here is a Kanban board, a visual tool for managing workflows and tracking tasks through stages like "To Do," "In Progress," and "Done." It helps teams quickly see the status of tasks and improve task flow.
+
+![](photos/10-view.jpg)
+
+The board consists of **Lists**. Each list represents a stage in the workflow. Typically, there are three lists: To Do, In Progress, and Done. However, you can rename, reorder, or add new lists by double-clicking on their names or dragging them.
+
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:158]: heading 4 of "Lists"

File: docs/main/03-research/01-vendors/04-trello/04-trello.md [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+Option | Description
+:- | :-
+Add card | Add a new card to the list.
+Copy list | Create a duplicate of the list by entering a new name and clicking Create list.
+Move list | Move the list to a different position.
+Move all cards in this list | Transfer all cards to another list.
+Sort by | Organize cards by date, name, etc.
+Automation | Set automation options for this list
+Archive | Archive the list, making it disappear from the board and moving it to the archived space.
+
```
Comment (atareversei) [inline docs/main/03-research/01-vendors/04-trello/04-trello.md:174]: heading 4 of cards

**Review by atareversei — CHANGES_REQUESTED** [review]

File: docs/main/03-research/01-vendors/04-trello/04-trello.md (hunk 6 above)
Comment (thisis-mahyar) [reply docs/main/03-research/01-vendors/04-trello/04-trello.md:146]: I added period to all types of the sentences (even short ones)

File: docs/main/03-research/01-vendors/04-trello/04-trello.md (hunk 9 above)
Comment (thisis-mahyar) [reply docs/main/03-research/01-vendors/04-trello/04-trello.md:158]: The heading hierarchy has became kind of messy here! For example, here the "Lists" has been assigend a h6! I recommend you to look them up @atareversei

**Review by atareversei — APPROVED** [review]

**Comment by thisis-mahyar** [comment]
done

**Pull request merged**